### DISCUSSION TITLE
Can you change the border width and color of a CTKLabel?

### DISCUSSION THREAD

**Discussion by 32-git** [TomSchimansky/CustomTkinter#432] [Q&A] [answered]
As the question above says, can you change the border width and color of a CTKLabel?

**Comment by 32-git** [comment, marked as answer]
Well I found a workaround which is using a CTKFrame and defining the border width and color there. Afterwards, simply place the CTKLabel in the frame (master=frame). 

Don't define the width of the label inside, that should be defined in the frame itself, so all there needs to be in the label is the master, text maybe and the corner radius.

**Reply by CruzerEXT** [reply]
Any example of your code, Sir?
